sklearn/_loss/tests/test_link.py

```diff
@@ -73,15 +73,15 @@ def test_link_inverse_identity(link):
         # we do not need to distinguish.
         raw_prediction = rng.normal(loc=0, scale=10, size=(n_samples))
 
-    assert_allclose(link.link(link.inverse(raw_prediction)), raw_prediction)
+    assert_allclose(link.link(link.inverse(raw_prediction)), raw_prediction, rtol=1e-2)
     y_pred = link.inverse(raw_prediction)
     assert_allclose(link.inverse(link.link(y_pred)), y_pred)
 
 
 @pytest.mark.parametrize("link", LINK_FUNCTIONS)
-def test_link_out_argument(link):
+def test_link_out_argument(link, global_random_seed):
```

Comment:
I'm not sure if we really need `global_random_seed` for this test. @marenwestermann What do you think?

Reply:
I agree with @lorentzenchr. Here we only want to test the "engineering" part that passing `out` is working as expected.

Reply:
Ok, then I'll revert the changes I made for this test.